t/t5403-post-checkout-hook.sh

```diff
@@ -16,10 +16,12 @@ test_expect_success setup '
 	git update-ref refs/heads/master $commit0 &&
```

Comment:
I am not even sure that that is a bug. Rather, I think it is by design, because the original way to have multiple worktrees with a single `.git/` directory was to cd to the worktree and set `GIT_DIR`.

Reply:
Done